project/Dependencies.scala

```diff
@@ -1,29 +1,61 @@
 import sbt._
+import sbt.Keys._
 
 object Dependencies {
 
+  private def ver(for211: String, for212: String) = Def.setting {
+    CrossVersion.partialVersion(scalaVersion.value) match {
+      case Some((2, 11)) => for211
+      case Some((2, 12)) => for212
+      case _             => sys.error("not good")
+    }
+  }
+
+  def monocle(module: String) = Def.setting {
```

Comment:
this will probably continue to be unused for gt-server (hopefully) I'd leave it out but meh.

Reply:
oh oops, good catch